frontend/src/components/ExternalLink.tsx

```diff
@@ -8,7 +8,9 @@ type Props = {
 };
 
 export default function ExternalLink({ to, className, children }: Props) {
-  const isHttpMode = window.location.protocol.startsWith("http");
+  const isHttpMode =
```

Comment:
can we extract this check now?

Reply:
Actually, I'm not sure we should do this and actually just use a NSIS installer, if that is not much extra work

Reply:
We decided to keep as-is. Since the check is more complicated now I think extracting it into `frontend/src/utils/isHttpMode.ts` would be good.

And we said we might want to do a hotfix. I will make a new milestone for this.

Reply:
Done!